.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "editor.formatOnSave": true,
+  "editor.formatOnSaveTimeout": 1500,
```

Comment:
Why a timeout?

Reply:
Valid question. Probably so that it doesn't try endlessly if something goes wrong. It's just from recommendations: https://www.cairo-lang.org/docs/quickstart.html#visual-studio-code-setup